packages/driver/src/multi-domain/cypress.ts

```diff
@@ -20,30 +20,24 @@ import { handleScreenshots } from './screenshots'
 
 const specBridgeCommunicator = new SpecBridgeDomainCommunicator()
 
-const setup = () => {
+specBridgeCommunicator.initialize(window)
+
+specBridgeCommunicator.once('initialize:cypress', ({ config, env }) => {
+  // eventually, setup will get called again on rerun and cy will
+  // get re-created
+  setup(config, env)
+})
+
+const setup = (cypressConfig: Cypress.Config, env: Cypress.ObjectLike) => {
   // @ts-ignore
   const Cypress = window.Cypress = $Cypress.create({
-    browser: {
-      channel: 'stable',
-      displayName: 'Chrome',
-      family: 'chromium',
-      isChosen: true,
-      isHeaded: true,
-      isHeadless: false,
-      majorVersion: 90,
-      name: 'chrome',
-      path: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
-      version: '90.0.4430.212',
-    },
-    defaultCommandTimeout: 4000,
-    execTimeout: 60000,
-    taskTimeout: 60000,
-    pageLoadTimeout: 60000,
-    requestTimeout: 5000,
-    responseTimeout: 30000,
-    // Set defaults to avoid warnings
-    waitForAnimations: true,
-    animationDistanceThreshold: 5,
+    ...cypressConfig,
+    env,
+    __isMultiDomain: true,
+    video: false, // never turn on video for multi-domain when syncing the config. This is handled in the primary.
```

Comment:
Suggest moving comment to above so it doesn't run into the comment below it
```suggestion
    // never turn on video for multi-domain when syncing the config. This is handled in the primary.
    video: false,
```

Reply:
updated in b7258276e6134ccda3642b3936e8330370b26f03